### PR TITLE
Copy edits

### PR DESCRIPTION
**Pull request by kblumenthal**
Basic copy edits for readability

### CODE REVIEW

File: readme.rst [hunk 1]
```diff
@@ -43,91 +30,57 @@ Try ``warcprox --help`` for documentation on command line options.
...
+
+In order to capture HTTPS traffic, warcprox acts as a "man-in-the-middle" (MITM). When it receives a ``CONNECT`` directive from a client, it generates a public key certificate for the requested site, presents to the client, and proceeds to establish an encrypted connection with the client. It then makes a separate, normal HTTPS connection to the remote site. It decrypts, archives, and re-encrypts traffic in both directions.
+
+Configuring a warcprox instance as a browser’s HTTP proxy will result in security certificate warnings because none of the certificates will be signed by trusted authorities. However, there is nothing malicious about warcprox functions. To use warcprox effectively, the client needs to disable certificate verification or add the CA certification generated by warcprox as a trusted authority. When using the latter, remember to undo this change when finished using warcprox.
```
Comment (nlevitt) [inline readme.rst]: "CA certification" should be "CA certificate"

File: readme.rst [hunk 2]
```diff
@@ -43,91 +30,57 @@ Try ``warcprox --help`` for documentation on command line options.
...
 ===
-For interacting with a running instance of warcprox.
+The warcprox API may be used to retrieve information from and interact with a running warcprox instance, including:
 
-* ``/status`` url
-* ``WARCPROX_WRITE_RECORD`` http method
-* ``Warcprox-Meta`` http request header and response header
+* Retrieving status information via ``/status`` URL
+* Writing WARC records via ``WARCPROX_WRITE_RECORD`` HTTP method
+* Querying and editing ``Warcprox-Meta`` HTTP request header and response header
```
Comment (nlevitt) [inline readme.rst]: "Querying and editing" doesn't totally make sense to me. Maybe something like "Controlling some warcprox settings per capture via the ``Warcprox-Meta`` HTTP request header". Not sure how to cover the response header in the same sentence fragment concisely and accurately, but I think it's ok to leave it out here.

File: readme.rst [hunk 3]
```diff
@@ -43,91 +30,57 @@ Try ``warcprox --help`` for documentation on command line options.
...
 
-1. while fetching url, calculate payload content digest (typically sha1)
-2. look up digest in deduplication database (warcprox supports a few different
-   ones)
-3. if found, write warc ``revisit`` record referencing the url and capture time
-   of the previous capture
-4. else (if not found),
+1. While fetching URL, calculate payload content digest (typically SHA1 checksum value)
+2. Look up digest in deduplication database (warcprox currently supports `sqlite <https://sqlite.org/>`_ by default, `rethinkdb <https://github.com/rethinkdb/rethinkdb>`_ with two different schemas, and `trough <https://github.com/internetarchive/trough>`_)
+3. If found, write warc ``revisit`` record referencing the url and capture time of the previous capture
```
Comment (nlevitt) [inline readme.rst]: Maybe capitalize WARC here to be consistent with some of your other edits

**Comment by kblumenthal** [comment]
Edits for readability updated in new commit as per https://github.com/internetarchive/warcprox/pull/95#discussion_r200491731

@nlevitt please go ahead and apply your < 80 lines retroactively and I'll refrain from that in future PRs.

**Comment by nlevitt** [comment]
Closing in favor of https://github.com/internetarchive/warcprox/pull/100

**Pull request closed without merging**